client/vault/README.md

```
@@ -0,0 +1,517 @@
+    # Contract rejected the transaction
+    print(f"TX failed: {e}")
+except ErrorWrapper.Rpc as e:
+    # RPC communication error
+    pass
+```
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)